measures/BuildExistingModel/measure.rb

```diff
@@ -257,6 +262,21 @@ def run(model, runner, user_arguments)
     end
     measures['BuildResidentialHPXML'][0]['simulation_control_run_period_calendar_year'] = args['simulation_control_run_period_calendar_year'].get if args['simulation_control_run_period_calendar_year'].is_initialized
 
+    # CO2 Emissions
+    if args['co2_emissions'].is_initialized
+      if args['co2_emissions'].get
+        cambium_gea = 'NWPPc' # FIXME: look this up based on xxx?
```

Comment:
@shorowit How do you suppose we should look up `cambium_gea` for each building ID?

We sample a "REEDS Balancing Area" for the national project, but not for the testing project.

Does "po_name" correspond to county name? I'm not sure if it's a 1-to-many mapping, but maybe we could use that?

Reply:
I assume that ResStock-HPXML should use the same lookups/data as is used in the [resstock-cambium](https://github.com/NREL/resstock-cambium) project. That doesn't necessarily mean using GEA nor even using the data files that happen to be provided by OS-HPXML. You should look at that project and get help from @nmerket and @ekpresent.

Reply:
I just came here to say what @joseph-robertson said about us sampling the REEDS Balancing Area in the national project. Maybe we should add that lookup to the testing project? I'm pretty sure there's a lookup from BA to GEA somewhere as needed.

Reply:
@nmerket I believe it would be pretty straightforward to add a REEDS Balancing Area tsv to the testing project. It just has a dependency on county.

Here are OS-HPXML's BA to GEA lookup and data files: https://github.com/NREL/OpenStudio-HPXML/blob/cambium/HPXMLtoOpenStudio/resources/data/cambium. Would these be the proper files to use, or something different that was used for resstock-cambium?

Reply:
@ejhw has circulated a BA->GEA lookup table in the past: [ba_to_gea.csv](https://github.com/NREL/resstock/files/7803268/ba_to_gea.csv)